QuestionAnswering/QA_ICL/backend/fastapi_app/README.md

```diff
@@ -116,6 +121,308 @@ Overview:
 - `GET` `/ontozeolite/zeolite-frameworks`: given criteria such as XRD peaks and unit cell dimensions, returns all zeolite frameworks that fulfil the requirements.
 - `GET` `/ontozeolite/zeolitic-materials`: given criteria such as framework code and an author's family name, returns all zeolitic materials that fulfil the requirements.
 
-### Frontend (to be deprectaed)
+### Frontend (to be deprecated)
```

Comment:
Remove this deprecated frontend section, as the info is at the top node

Reply:
Could you clarify what you mean by "the info is at the top node"?

Reply:
Oh I mean there is no need to include any frontend details here as this is the backend docs. The specific docs are already available somewhere in the frontend or the overall architecture docs. Removing the section should be fine